utils/jwtGeneration/jwtHelper.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable camelcase*/
+const KJUR = require("jsrsasign");
+const fs = require("fs");
+const JSONWebKey = require("json-web-key");
+const jose = require("node-jose");
+const JWK = jose.JWK;
+const JWE = jose.JWE;
+const { find, keys, assign, flow, map } = require("lodash/fp");
+const yaml = require("js-yaml");
+const { refineMetadata } = require("../refineMetadata");
+
+const getKeyByUse = (json, useCase) =>
+  flow(
+    keys,
+    map(kid => assign(json[kid], { kid })),
+    find(keyObject => keyObject.use === useCase)
+  )(json);
+
+const keysYaml = yaml.safeLoad(
+  fs.readFileSync("./utils/jwtGeneration/keys.yml", "utf8")
+);
+const keysJson = JSON.parse(JSON.stringify(keysYaml, null, 4));
+
+module.exports.generateToken = function(questionnaireId, metadata) {
+  const signingKeyObject = getKeyByUse(keysJson.keys, "signing");
+  const encryptionKeyObject = getKeyByUse(keysJson.keys, "encryption");
+
+  // Header
+  const oHeader = {
+    alg: "RS256",
```

Comment:
Could we lift this and the value below as constants with nicer names. Especially as `RS256` is reused below.